cvat/settings/base.py

```diff
@@ -189,17 +189,19 @@ def generate_ssh_keys():
 
 # Cross-Origin Resource Sharing settings for CVAT UI
 UI_SCHEME = os.environ.get('UI_SCHEME', 'http')
-UI_HOST = os.environ.get('UI_HOST', 'localhost')
-UI_PORT = os.environ.get('UI_PORT', '3000')
-CORS_ALLOW_CREDENTIALS = True
-CSRF_TRUSTED_ORIGINS = [UI_HOST]
-UI_URL = '{}://{}'.format(UI_SCHEME, UI_HOST)
-
-if UI_PORT and UI_PORT != '80':
-    UI_URL += ':{}'.format(UI_PORT)
-
-CORS_ORIGIN_WHITELIST = [UI_URL]
-CORS_REPLACE_HTTPS_REFERER = True
+UI_HOST = os.environ.get('UI_HOST')
```

Comment:
Why do we need to remove values by default?

Reply:
Otherwise we will have to specify twice the same value for CVAT_HOST and UI_HOST that used for redirects by django backend, It's not good from my point of view.
In this patch I use the same approach as for UI server: default value is not specified and all urls will be relative, but these values are specified for the dev environments.
https://github.com/opencv/cvat/blob/3d4e7268e175557543932de115e0d7c03fae9e43/cvat-ui/package.json#L8

Reply:
@azhavoro , let's try to  move the code to development.py configuration file.

Reply:
@nmanovic Great idea! Done.